translations/src/config.js

```diff
@@ -68,18 +69,53 @@ class Config {
     return Object.keys(this.supportedLanguages)
   }
 
-  getRTLLanguages (): string[] {
-    return entries(this.supportedLanguages)
-      .filter(([_, languageConfig]) => languageConfig.rtl)
-      .map(([languageCode, _]) => languageCode)
+  getSupportedLanguage (languageCode: string): LanguageType | void {
+    const fallbacks = this.fallbacks[languageCode]
+
+    if (fallbacks) {
+      const found = fallbacks.find(fallback => !!this.supportedLanguages[fallback])
+
+      if (found) {
+        return this.supportedLanguages[found]
+      }
+    }
+
+    return this.supportedLanguages[languageCode]
+  }
+
+  isSupportedLanguage (languageCode: string): boolean {
+    return !!this.getSupportedLanguage(languageCode)
   }
 
-  isRTLLanguage (languageCode: string): boolean {
-    return this.getRTLLanguages().includes(languageCode)
+  /**
+   * Checks whether the languageCode belongs has an RTL script. This decision is made by the project "Integreat".
+   * Writing direction is not an attribute of "language", but of "scripts". That means that there are language which
+   * can have RTL and LTR scripts.
```

Comment:
Are there really? If there are and we use one of these languages, this makes not really sense I think. Why do we show a language with both rtl and ltr scripts as rtl? I also don't really see a point in having to scripts for one language in the apps as mixing is a really bad idea, so we should just decide for each language whether to use the rtl or ltr script.

Reply:
We are deciding for each language whether we think that our uses have a ltr/rtl script.

It is very complicated to get to a list of language -> direction. In fact there is no such list. The linked article states this. We have to decide for our "Integreat" app specificly whether we thinkg users are rtl/ltr.